dangerzone/gui/main_window.py

```diff
@@ -237,11 +236,53 @@ def __init__(self, dangerzone: DangerzoneGui) -> None:
         layout.addWidget(self.doc_selection_widget, stretch=1)
         self.setLayout(layout)
 
-    def documents_selected(self, selected_docs: List[Document]) -> None:
-        self.doc_selection_widget.hide()
-        self.settings_widget.show()
+    def documents_selected(self, new_docs: List[Document]) -> None:
+        if not self.conversion_started:
+
+            # assumed all files in batch are in the same directory
+            first_doc = new_docs[0]
+            output_dir = os.path.dirname(first_doc.input_filename)
+            if not self.dangerzone.output_dir:
+                self.dangerzone.output_dir = output_dir
+            elif self.dangerzone.output_dir != output_dir:
+                Alert(
+                    self.dangerzone,
+                    message="Dangerzone does not support adding documents from multiple locations.\n\n The newly added documents were ignored.",
+                    has_cancel=False,
+                ).exec_()
+                return
+            else:
+                self.dangerzone.output_dir = output_dir
```

Comment:
During my tests, I saw that users can add documents from various directories with:

```
$ dangerzone dir1/doc.pdf dir2/doc.pdf
$ dangerzone dir1/doc.pdf dir2/doc.pdf
```

With the GUI, I can also save the files to a third directory. So, what do we do there? Should we have parity across platforms and catch this case as well, in the GUI and CLI, or let it be an allowed, but undefined behavior.

Personally, I'd lean on taking the less complicated approach, as we're nearing to the 0.4.0 release. But we can discuss.